packages/clay-select/README.md

```diff
@@ -0,0 +1,26 @@
+# clay-select
+
+Clay Select
+
+## Setup
+
+1. Install NodeJS >= v0.12.0 and NPM >= v3.0.0, if you don't have it yet. You
+can find it [here](https://nodejs.org).
+
+2. Install local dependencies:
+
+  ```
+  npm install
+  ```
+
+3. Build the code:
+
+  ```
+  npm run build
+  ```
+
+4. Open the demo at demos/index.html on your browser.
+
+## Contribute
+
+We'd love to get contributions from you! Please, check our [Contributing Guidelines](CONTRIBUTING.md) to see how you can help us improve.
```

Comment:
I'd like to see the API documented in `README.md`. Users should not need to parse the source (`STATE`, `demos/index.html`) to see how to use this.